README.md

```diff
@@ -15,8 +15,8 @@ Fluent UI Apple contains native UIKit and AppKit controls aligned with [Microsof
 #### Requirements
 
 - iOS 14+ or macOS 10.15+
```

Comment:
The build settings for the Mac app says 12.3, but the xcconfig file says 10.15. Do we need to align those? Doesn't have to happen in this PR, either way.

Reply:
That's a good question! I'm not quite sure... I can make those changes if need be. @mischreiber?